$NetBSD$

Install as "dosbox-staging" to avoid a conflict with emulators/dosbox.

--- contrib/linux/dosbox-staging.desktop.orig	2021-07-02 15:47:39.000000000 +0000
+++ contrib/linux/dosbox-staging.desktop
@@ -55,7 +55,7 @@ GenericName[wa]=Èn emulateu DOS
 GenericName[zh_CN]=DOS 模拟器
 GenericName[zh_TW]=DOS 模擬器
 Comment=DOS/x86 emulator focusing on ease of use
-Exec=dosbox
+Exec=dosbox-staging
 Icon=dosbox-staging
 Type=Application
 Terminal=false
